pkg/server/server.go

```diff
@@ -109,6 +110,9 @@ func startWrangler(ctx context.Context, config *Config) (string, error) {
 		return "", err
 	}
 
+	// start the node cache
+	nodeCache := sc.Core.Core().V1().Node().Cache()
```

Comment:
Is it possible to revert the changes to `setMasterRoleLabel` and just call `sc.Core.Core().V1().Node().Cache()`?